package/scripts/windows/quarto.cmd

```diff
@@ -21,34 +28,44 @@ IF EXIST "!QUARTO_TS_PATH!" (
 		GOTO end
 	)
 
-  call !QUARTO_ROOT!\win_configuration.bat
   REM overrride share path to point to our dev folder instead of dist
-  set QUARTO_SHARE_PATH=!QUARTO_ROOT!\src\resources
+  set "QUARTO_SHARE_PATH=!QUARTO_ROOT!\src\resources"
 
 	IF NOT DEFINED QUARTO_ACTION (
 		SET QUARTO_ACTION=run
 	)
-	SET "QUARTO_IMPORT_ARGMAP=--importmap=""!QUARTO_SRC_PATH!\dev_import_map.json"""
+	SET "QUARTO_IMPORT_ARGMAP=--importmap=""!QUARTO_ROOT!\src\dev_import_map.json"""
 
 	IF NOT DEFINED QUARTO_TARGET (
-		SET "QUARTO_TARGET=!QUARTO_TS_PATH!"
+		SET "QUARTO_TARGET=!QUARTO_ROOT!\src\quarto.ts"
 	)
 
+  REM Reads the configuration file line by line
+  REM and convert any export statements into set statements
+  REM (allows reuse of variables)
+
+  FOR /F "tokens=*" %%A IN (!QUARTO_ROOT!\configuration) DO (
+    SET LINE=%%A
+
+    IF "!LINE:~0,7!"=="export " (
+      SET "LINE=!LINE:export =!"
+      SET "!LINE!"
+    )
+  )
+
 	SET QUARTO_DEBUG=true
 	:: Normalize path to remove ../.. stuff
-	for %%i in ("!SCRIPT_PATH!..\config\deno-version") do SET "DENO_VERSION_FILE=%%~fi"
+	for %%i in ("!SCRIPT_PATH!\..\config\deno-version") do SET "DENO_VERSION_FILE=%%~fi"
 
   if exist "!DENO_VERSION_FILE!" (
 
 		set /p DENO_INSTALLED_VERSION=<"!DENO_VERSION_FILE!"
 	  if NOT "!DENO!"=="!DENO_INSTALLED_VERSION!" (
 			echo !DENO!>"!DENO_VERSION_FILE!"
-
-			cd !QUARTO_ROOT!
-			call configure.cmd
-      echo
-			echo Quarto required reconfiguration to install Deno !DENO!. Please try command again.
-			GOTO end
+      echo "Quarto needs reconfiguration to install the correct Deno version (!DENO!)."
+      echo "    (Found Deno version: !DENO_INSTALLED_VERSION!)"
+      echo "Please run the configure.sh script in either git bash or MSYS2."
```

Comment:
I found that calling "bash configure.sh" ended up having a hard time with windows line endings and failing pretty horribly. Exiting and telling the developer to use MSYS2 or git bash seemed like a better option than trying to get that to work.

Reply:
Did not looked at the context for this, but just for reference, when you have WSL installed, calling `bash` will mean running in WSL not in a Windows bash emulator. Which makes things pretty different. Anyway, I'll test this.

Reply:
Yes, that's exactly what happens for me, and that's where the line ending stuff arises. WSL doesn't compensate for it, but the MSYS2/git bash stuff does. Or at least that's my working theory...

Reply:
With WSL the script will be running on Ubuntu. Not Windows, so it is not just the Line ending that change. All the downloads and installation won't work I think in that case. We are calling powershell I think to download on Windows even from the sh script, so it definitely should not run under bash for WSL.

Reply:
It wasn't my intent to get that working. I consider WSL to be truly "linux" and it shouldn't be taking the windows path in these scripts at all. Perhaps I messed up that case. I was developing only with git bash and msys2. Is it enough to have it documented as it is now, or should we try to either support it or fail more gracefully in the case where WSL bash gets used?

Reply:
Yes, this is completely true, and we should not ever call bash from a .cmd script. You comment elsewhere about the reconfiguration stuff, and I'll answer more there. Suffice to say, this is a code path that is not handled very well in this PR, and yes, it is possible as it stands to call bash from cmd or powershell, resulting in a confusing failure to run the .sh script.